### ISSUE TITLE
MfxNotification* component increase startup time

### ISSUE THREAD

**Issue by TheMesoria** [palexdev/MaterialFX#317]
**Describe the bug**
`MfxNotification*` systems are incredibly cumbersome and have a HEAVY impact on startups. To simplify it, I will show profile outputs from the same application startup (in my case, notifications are unnecessary for startups).

![image](https://user-images.githubusercontent.com/9269398/236689963-4955668a-7dfd-4d0c-a28d-7420f80e3ae0.png)

The boot time of the application WITHOUT this system is less than 1 sec. With the `MfxNotification*` component, it's around 7-10 sec on a pretty... sufficient machine. On the target machine, it takes even up to 1 min! This is insane and is not visible on a demo project.

My project startup:

18:42:55.851 [JavaFX Application Thread] Start
18:43:03.194 [JavaFX Application Thread] Done

Demo project startup:

2023/05/07 18:54:26 Start                    // DemoController.java:101
2023/05/07 18:54:27 End                     // DemoController.java:129

Empty project with this snippet:

(snippet 1)
```java
        Platform.runLater(() -> {

            DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss");

            System.out.println(dtf.format(LocalDateTime.now()));
            MFXNotificationSystem.instance().initOwner(primaryStage);
            MFXNotificationCenterSystem.instance().initOwner(primaryStage);
            System.out.println(dtf.format(LocalDateTime.now()));

            MFXNotificationCenter center = MFXNotificationCenterSystem.instance().getCenter();
            center.setCellFactory(notification -> new MFXNotificationCell(center, notification) {
                {
                    setPrefHeight(400);
                }
            });
        });
```

2023/05/07 18:49:45
2023/05/07 18:49:53

I might be missing something, but with a quick sneak-peak into the documentation, I could not find the step I missed. I acknowledge that I am unaware of everything inside this wonderful project. From what I understand, Looking at a specific controller should be sufficient to know how the control works.

It's worth noting that overall, the `MfxDemo` boot time is also kind of.. sloopy (6-8 sec, which fits my time frame; not necessarily because of the same issue, just time correlates).  The longes part is `primaryStage.show();` (4sec), which is not surprising, but I commented out all controllers out of `MfxLoader` (commented out `private void initializeLoader()`). 

A couple of points worth noting:

- If I don't use the `MfxNotification*` systems, I can get boot times of under 1 sec; once I insert snippet no.1, the program takes 6-10 sec to boot. On slower machines, it takes around 1 min, which is unacceptable)
- I am using other controls (buttons, checkboxes, etc.) and styling them without issues
- MRE causes the same issues event without notification popping.

**[MRE](https://bit.ly/3k5gTJf)(Minimal Reproducible Example)**

MRE: [untitled.zip](https://github.com/palexdev/MaterialFX/files/11415565/untitled.zip)

```java
    @Override
    public void start(Stage primaryStage) {

        Platform.runLater(() -> {

            DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss");

            System.out.println(dtf.format(LocalDateTime.now()));
            MFXNotificationSystem.instance().initOwner(primaryStage);
            MFXNotificationCenterSystem.instance().initOwner(primaryStage);
            System.out.println(dtf.format(LocalDateTime.now()));

            MFXNotificationCenter center = MFXNotificationCenterSystem.instance().getCenter();
            center.setCellFactory(notification -> new MFXNotificationCell(center, notification) {
                {
                    setPrefHeight(400);
                }
            });
        });

        primaryStage.setOnShown(e -> Platform.exit());
        primaryStage.show();
    }
```

**To Reproduce**
1. Should be visible on the start

Worth mentioning:
- Tested on oJdk-19, oJdk-20

**Expected behavior**
Somewhat decent startup time

**Comment by palexdev** [comment]
@TheMesoria thanks for the analysis
Given that the notification systems seem to give me more troubles than benefits I think I'll remove them by the next version 

The culprit of this could be the fact that each notification system create a new separate stage that allow the popups to show, since they need a parent 
Creating windows is a costly operation, but I didn't expect it to be this much. Also like I said there are other issues like #213.
Even worse on my KDE system, which uses effects to blur the windows, I can see the windows that supposedly should be hidden! Yep, I'm definitely going to remove them

**Comment by palexdev** [comment]
Fixed in version `11.17.0`
